docs/decisions/0002-mark-fuzzy-entries.rst

```diff
@@ -0,0 +1,82 @@
+Mark Transifex invalid entries as fuzzy
+#######################################
+
+Problem
+*******
+
+As part of the `OEP-58`_ Transifex GitHub App bot has now syncs Translations.
+Bot creates pull requests with invalid entries in the
+po-files. These entries are validated by `LexiQA`_, Transifex-builtin quality
+checks as well as human reviewers. However, some invalid translations still
+make it into the sync translation pull requests which breaks
+``django compilemessages`` command and is catchable by ``msgfmt``.
+
+If Transifex had marked the invalid entries as fuzzy, then the
+``django compilemessages`` command would have ignored them. However, that's
+not the case. Therefore, we need to perform this task ourselves.
+
+
+Design Decision
+***************
+
+Implement a GitHub Actions workflow that will mark the invalid entries as
+``fuzzy``. Then create a commit -- editing the pull request -- to branch
+of the bot pull request.
+
+To ensure a safe and reliable workflow, the following workflows will be
+combined into one single workflow with multiple jobs:
+
+#. Run ``python-tests.yml`` to validate the python code
+#. Then, run ``validate-translation-files.yml`` which performs the following:
+
+   #. Validate the po-files using ``msgfmt``
+   #. Mark the invalid entries as fuzzy (writes to the pofile)
+   #. Validate the files again to ensure the files are became valid
+   #. Commit the fixes if any
+
+#. Mark invalid entires as fuzzy therefore excluded from .mo files as well as msgfmt
+#. Validate the files again to ensure the files are still valid
+#. Commit the fixes
+#. Run the lint commits
+#. Push to the branch
+#. Trigger auto merge
+
+**Pros**
+
+* There's no need for manual intervention, therefor it's fast and won't
+  create a backlog of pull requests.
```

Comment:
Avoiding a PR backlog is one pro, but I don't think it's the only one. I think I touched on a couple in my comment on slack.

> There are a couple of use cases that come to mind (please let me know if I'm misunderstanding this):
> * Old translated string passes msgfmt, new translated string fails msgfmt
>   * With the current process, the old string would still be pulled by atlas
>   * With the fuzzy process, the new fuzzy string would be pulled by atlas and the string wouldn't be translated in the application
> * Lots of new strings added, one fails msgfmt
>   * With the current process, none of the new strings would make it into the openedx-translations repo
>   * With the fuzzy process, all of the new strings would make it in